default/views.py

```
@@ -1,3 +1,87 @@
+from django.shortcuts import get_object_or_404
+from .models import Post, Comment
+from .serializers import (
+    PostListSerializer, PostDetailSerializer, PostCreateUpdateSerializer,
+    CommentSerializer, CommentCreateSerializer
+)
 
-# Create your views here.
+
+class PostListCreateView(generics.ListCreateAPIView):
+    queryset = Post.objects.all()
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)